switools/signaturelib.py

```diff
@@ -12,4 +14,45 @@ def getSigFileName( swiFile ):
       return SWIX_SIG_FILE_NAME
    return SWI_SIG_FILE_NAME
 
+def runCmd( cmd, workDir ):
+   try:
+      subprocess.check_call( cmd.split(" "), cwd=workDir )
```

Comment:
Pass `cmd` as a list so you don't split on the wrong spaces e.g. spaces in filenames.  Also, maybe add a default value of `None` for `workDir`?